metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/dateextractors/EdtfDateExtractorTest.java

```diff
@@ -0,0 +1,319 @@
+package eu.europeana.normalization.dates.extraction.dateextractors;
+
+import static eu.europeana.normalization.dates.edtf.DateEdgeType.OPEN;
+import static eu.europeana.normalization.dates.edtf.DateEdgeType.UNKNOWN;
+import static eu.europeana.normalization.dates.edtf.DateQualification.APPROXIMATE;
+import static eu.europeana.normalization.dates.edtf.DateQualification.NO_QUALIFICATION;
+import static eu.europeana.normalization.dates.edtf.DateQualification.UNCERTAIN;
+import static eu.europeana.normalization.dates.edtf.DateQualification.UNCERTAIN_APPROXIMATE;
+import static eu.europeana.normalization.dates.edtf.IntervalEdtfDate.DATES_SEPARATOR;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.of;
+
+import eu.europeana.normalization.dates.DateNormalizationResult;
+import eu.europeana.normalization.dates.DateNormalizationResultStatus;
+import eu.europeana.normalization.dates.edtf.AbstractEdtfDate;
+import eu.europeana.normalization.dates.edtf.InstantEdtfDate;
+import eu.europeana.normalization.dates.edtf.IntervalEdtfDate;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class EdtfDateExtractorTest {
+
+  private final EdtfDateExtractor edtfDateExtractor = new EdtfDateExtractor();
+
+  // TODO: 01/03/2023 Possible reuse of the test code here for all extractors
+  private void extract(String input, String expected) {
```

Comment:
sggestion: rename to `assertEdtfDateExtractor`

Reply:
Done. For consistency I've kept it similar to other tests.